test/e2e/specs/adding-blocks.test.js

```diff
@@ -84,7 +81,8 @@ describe( 'adding blocks', () => {
 		// Using the between inserter
 		const insertionPoint = await page.$( '[data-type="core/quote"] .editor-block-list__insertion-point-button' );
 		const rect = await insertionPoint.boundingBox();
-		await page.mouse.move( rect.x + ( rect.width / 2 ), rect.y + ( rect.height / 2 ) );
+		await page.mouse.move( rect.x + ( rect.width / 2 ), rect.y + ( rect.height / 2 ), { steps: 10 } );
```

Comment:
I saw this `{ steps: 10 }` argument across the tests though; if they need to stay *and* you don't want to add the eslint rule now, please file a follow-up issue for it as I think it's important 😄

Reply:
I'll second the worry about `steps`. It's not entirely clear to me why they're necessary (aside from the obvious "it makes the tests green").

Is it a bug with Puppeteer? If so, we should file an issue.
Is it our application requiring multiple mousemove events to register that the effect is to take place? If so, we should know about this and consider whether it's a bug or intentional.

How also do we communicate to developers if and when they'll need to provide this in their tests? This may speak to @tofumatt 's point about an ESLint rule.

But I'd rather we first have a good understanding of why it's here.

Reply:
Removing the `steps` call reliably caused many of my tests to fail though, so it seems they're needed. I just can't see why from the code.